addon/webextension/background/main.js

```diff
@@ -55,6 +55,17 @@ window.main = (function () {
     catcher.watchPromise(loadSelector());
   }));
 
+  browser.tabs.onUpdated.addListener((id, info, tab) => {
+    if (info.url && tab.selected) {
+      if (/^(?:about|data|moz-extension):/.test(info.url)) {
```

Comment:
We do have behavior for about:new and about:blank (opening the shots page).  So we need a match to treat those separately.